src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -288,8 +291,11 @@ export default function Page() {
                     </> : <></>}
                 <DateSelectorCard onDateChange={updateDateRange} date={selectedDate} />
                 
+                
             </HStack>
             {useMemo(() => renderWarning(), [selectedDate])}
+        <div>
```

Comment:
do we need the disabled field for this ticket?

Reply:
I believe this is in the ticket because the disabled field was already merged in.